### PR TITLE
main/powertop: update to 2.10

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: main/powertop/APKBUILD [hunk 1]
```diff
@@ -24,11 +20,15 @@ build() {
 	make
 }
 
+check() {
+	# There is no testsuite, check the binary
+	./src/powertop --version
+}
+
```
Comment (clandmeter) [inline main/powertop/APKBUILD]: We have had discussions about this before and we prefer not to use a simple test like this.
Check function should always use a testsuite or else disable it. We dont want to give the impression
we test our packages but instead just do a simple version check.

Comment (maxice8) [reply main/powertop/APKBUILD]: Can you update https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#options !check entry to reflect this change in policy ?

Comment (maxice8) [reply main/powertop/APKBUILD]: Updated. Thanks for the review.

**Comment by ncopa** [comment]
Checksums needs to be fixed

**Comment by ncopa** [comment]
This is a bug:
````
event-parse.c:5124:7: warning: assignment to 'const char *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
   msg = strerror_r(errnum, buf, buflen);
       ^         
````

It seems to expect GNU libc `strerror_r` behavior. It needs to be fixed to use POSIX variant, which returns `0` for success and `-1` for fail.

**Pull request closed without merging**